solc/CommandLineInterface.cpp

```diff
@@ -594,6 +594,26 @@ bool CommandLineInterface::compile()
 
 		m_compiler->enableIRGeneration(m_options.compiler.outputs.ir || m_options.compiler.outputs.irOptimized);
 		m_compiler->enableEwasmGeneration(m_options.compiler.outputs.ewasm);
+		m_compiler->enableEvmBytecodeGeneration(
+			m_options.compiler.estimateGas ||
```

Comment:
Not needed after https://github.com/ethereum/solidity/pull/11903 if experimental via ir is activated...